src/MolangData/MolangFunction.ts

```diff
@@ -4,6 +4,10 @@ import { Types } from "bc-minecraft-bedrock-types";
  *
  */
 export interface MolangFunction extends Types.Identifiable, Types.Documentated {
+
+  id : string;
+
+  documentation: string;
```

Comment:
These are already in the Types.Identifiable and Types.Documentated 🤔

Reply:
I didn't see them in the autocomplete so I added them

Reply:
Nw